tests/ValidatorTest.php

```diff
@@ -862,39 +862,39 @@ public function initSkipOnEmptyDataProvider(): array
             'null' => [
                 null,
                 new class () {
-                    #[Required]
```

Comment:
We can leave / add one test with required to make sure this check works:

https://github.com/yiisoft/validator/blob/b77dedd843d0157e8be8879dab811a68d17bc7a5/src/Validator.php#L198-L200

Reply:
Agree, but PR about another. Suggest make this later, when we will be increase psalm to 1 and kill mutants.

Reply:
OK. Let's do it in a separate PR.